pkg/materialize/identifiers.go

```diff
@@ -14,15 +14,11 @@ func GetIdentifierSchemaStruct(databaseName string, schemaName string, v interfa
 	if v, ok := u["name"]; ok {
```

Comment:
Should this function even take `databaseName` or `schemaName` as input now? They look unused now.